src/frontend/apps/e2e/__tests__/app-desk/menu.spec.ts

```diff
@@ -2,9 +2,9 @@ import { expect, test } from '@playwright/test';
 
 import { keyCloakSignIn } from './common';
 
-test.beforeEach(async ({ page }) => {
+test.beforeEach(async ({ page, browserName }) => {
   await page.goto('/');
-  await keyCloakSignIn(page);
+  await keyCloakSignIn(page, browserName);
```

Comment:
it quite confusing, to add a `browserName` param to the `keycloakSignIn` function